# gcc64: from live-bootstrap b1ceced7, steps/musl-1.1.24/patches/makefile.patch, byte-identical below
# this block (which GNU patch skips as leading text).  Why the chain needs it:
# `tcc -ar` cannot create empty archives (libm.a, libpthread.a, ...).
SPDX-FileCopyrightText: 2021 Andrius Štikonas <andrius@stikonas.eu>
SPDX-FileCopyrightText: 2021 Samuel Tyler <samuel@samuelt.me>

SPDX-License-Identifier: MIT

tcc -ar does not support creating empty archives
--- musl-1.1.24/Makefile	2019-10-13 22:58:27.000000000 +0100
+++ musl-1.1.24/Makefile	2021-02-01 00:21:14.974687663 +0000
@@ -167,7 +167,7 @@
 
 $(EMPTY_LIBS):
 	rm -f $@
-	$(AR) rc $@
+	touch $@
 
 lib/%.o: obj/crt/$(ARCH)/%.o
 	cp $< $@
